src/main/java/mate/academy/hibernate/relations/service/impl/CountryServiceImpl.java

```diff
@@ -1,16 +1,25 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.impl.CountryDaoImpl;
```

Comment:
It's a good practice to depend on abstractions rather than concrete implementations. Consider using the CountryDao interface instead of the concrete CountryDaoImpl class.